cvxpy/settings.py

```diff
@@ -174,12 +174,13 @@
 UNKNOWN = "UNKNOWN"
 
 # Canonicalization backends
+STACKED_SLICES_BACKEND = "STACKED"
```

Comment:
If we find that this is always at least as fast as the regular scipy backend, we might even replace that one.